src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -728,6 +732,10 @@ public VariantContextWriter createVCFWriter(final File outFile) {
             }
         }
 
+        if (sitesOnlyMode) {
+            options.add(Options.DO_NOT_WRITE_GENOTYPES);
```

Comment:
Can you check to see whether there are any other `GATKTools` that bypass `GATKTool.createVCFWriter()` when writing a VCF and call `GATKVariantContextUtils.createVCFWriter()` directly instead? (besides the ones you dealt with below)

Don't worry about non-GATKTools.

Reply:
I have already checked. It is bypassed in a number of tests, the HaplotypeCaller, and `CreateSomaticPanelOfNormals` which doesn't extend `GATKTool` so I elected not to include it here.